images/windows/scripts/docs-gen/Generate-SoftwareReport.ps1

```diff
@@ -20,7 +20,12 @@ Import-Module (Join-Path $PSScriptRoot "SoftwareReport.VisualStudio.psm1") -Disa
 # Software report
 $softwareReport = [SoftwareReport]::new($(Build-OSInfoSection))
 $optionalFeatures = $softwareReport.Root.AddHeader("Windows features")
-$optionalFeatures.AddToolVersion("Windows Subsystem for Linux (WSLv1):", "Enabled")
```

Comment:
nit. something like below might be better
```
$optionalFeatures.AddToolVersion("Windows Subsystem for Linux (WSLv1):", "Enabled")
 if (Test-IsWin25) {
    $optionalFeatures.AddToolVersion("Windows Subsystem for Linux (Default, WSLv2):", $(Get-WSL2Version))
}
```

Reply:
Updated